docs/src/administration/users.md

```diff
@@ -1,65 +1,53 @@
 ---
 title: "User administration"
 weight: 1
-sidebarTitle: "Users"
+sidebarTitle: "User administration"
 description: |
-  Every Platform.sh user has a role that controls their access and permission levels. Different roles allow different levels of access to your applications, environments and projects. You can manage how users interact with your project and environments at Platform.sh
+  Every Platform.sh user has a set of permissions. Depending on your role, you'll be able to access different levels of the application, environments and projects. If you want to add a user to a project or an environment type, the user has to create an account before they can contribute to the project.
```

Comment:
This topic is about administering users. Given that, the reader is not going to have different levels of access. They will have to be an admin or owner. So the bit about "depending on your role" seems confusing to me. 

I'd also recommend shortening this as it's rather long for a `description`. Some of the detail doesn't need to be here, but can be below. If you like, you can separate the description from the intro text.

I think the bit about accessing different levels of the application is somewhat confusing. At least it should have a serial comma. :slightly_smiling_face:

Reply:
Ok